src/EnergyPlus/Plant/Solution/PlantLoopSolver.cc

```diff
@@ -1005,18 +1003,14 @@ namespace PlantLoopSolver {
 			ShowFatalError( "Program terminates due to preceding conditions." );
 		}
 
-	}
+        CurrentLoopSideIsConstantSpeedBranchPumped = ThisLoopHasConstantSpeedBranchPumps( ThisSide );
 
-	//==================================================================!
-	//==================================================================!
-	//==================================================================!
+        return LoopFlow;
 
-	//==================================================================!
-	//================== LOOPSIDE BRANCH SIMULATION ====================!
-	//==================================================================!
+	}
 
 	void
-	SimulateAllLoopSideBranches(
+	PlantLoopSolverClass::SimulateAllLoopSideBranches(
```

Comment:
Oh, and I pulled all of these functions into methods on a struct.  I didn't really touch the guts of each method, so this is an incremental improvement.  Eventually, we could have each plant loop properly store its own solver instance, and reduce the number of arguments being passed around to each function and the amount of global data required to test.